README.md

```diff
@@ -280,6 +280,7 @@ Create postgres DB and S3 with development settings:
 
     cf create-service aws-rds shared-psql crt-db
     cf create-service s3 basic-public crt-s3
+    cf create-service s3 basic sso-creds
```

Comment:
I think now that there will be data beyond the SSO creds stored in this bucket, we should rename it to something more generic (e.g., `crt-private`) to avoid confusion. I will document this as a standalone card

Reply:
Thanks!